internal/wc.go

```diff
@@ -0,0 +1,33 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func Wc()(lineCount,wordCount,bytesCount int, err error) {
+	filepath := os.Args[1]
+	content , err := ScanFile(filepath)
+
+	lines := strings.Split(content, "\n")
+	lineCount = 0
+	wordCount = 0
+	bytesCount = 0
+	if err != nil {
+		fmt.Print(0,0,0)
+		return 0, 0, 0 , err
+	}
+	for _,line:= range lines {
+		lineCount++
+		bytesCount += len(line)
+		words := strings.Fields(line)
+		wordCount += len(words)
```

Comment:
no need to define new variable

```suggestion
		wordCount += len(strings.Fields(line))
```

Reply:
Alright